tests/test_tle_predictor.py

```diff
@@ -137,85 +138,49 @@ class SubPredictedPass(PredictedPass):
+        pass_ = self.predictor.get_next_pass(ARG, date, max_elevation_gt=15)
         for i in range(20):
-            pass_ = self.predictor.get_next_pass(tortu1, pass_.los, max_elevation_gt=15)
+            pass_ = self.predictor.get_next_pass(ARG, pass_.los, max_elevation_gt=15)
             self.assertGreaterEqual(pass_.max_elevation_deg, 15)
 
     def test_get_next_pass_with_limit_exception(self):
         date = datetime.datetime.strptime("2014-10-22 20:18:11.921921", '%Y-%m-%d %H:%M:%S.%f')
-        pass_ = self.predictor.get_next_pass(tortu1, date, max_elevation_gt=15)
+        pass_ = self.predictor.get_next_pass(ARG, date, max_elevation_gt=15)
         with self.assertRaises(NotReachable):
-            self.predictor.get_next_pass(tortu1, date, max_elevation_gt=15,
+            self.predictor.get_next_pass(ARG, date, max_elevation_gt=15,
                                          limit_date=pass_.aos - datetime.timedelta(minutes=1))
 
     def test_get_next_pass_with_limit(self):
         date = datetime.datetime.strptime("2014-10-22 20:18:11.921921", '%Y-%m-%d %H:%M:%S.%f')
-        pass_ = self.predictor.get_next_pass(tortu1, date, max_elevation_gt=15)
+        pass_ = self.predictor.get_next_pass(ARG, date, max_elevation_gt=15)
         new_pass = self.predictor.get_next_pass(
-            tortu1, date, max_elevation_gt=15,
+            ARG, date, max_elevation_gt=15,
             limit_date=pass_.los + datetime.timedelta(seconds=1))
         self.assertEqual(pass_, new_pass)
 
     def test_get_next_pass_while_passing(self):
-        date = datetime.datetime.strptime("2014/10/23 01:27:10", '%Y/%m/%d %H:%M:%S')
-        pass_ = self.predictor.get_next_pass(tortu1, date)
-        self.assertEqual(pass_.aos, date)
+        date = datetime.datetime.strptime("2014/10/23 01:27:33.224", '%Y/%m/%d %H:%M:%S.%f')
+        pass_ = self.predictor.get_next_pass(ARG, date)
+        self.assertAlmostEqual(pass_.aos, date, delta=ONE_SECOND)
         self.assertTrue(date < pass_.los)
 
         position = self.predictor.get_position(date)
-        self.assertTrue(tortu1.is_visible(position))
+        self.assertTrue(ARG.is_visible(position))
 
-    def test_grater_than_deg(self):
+    def test_greater_than_deg(self):
         date = datetime.datetime.strptime("2014/10/23 01:25:09", '%Y/%m/%d %H:%M:%S')
-        pass5 = self.predictor.get_next_pass(tortu1, date, aos_at_dg=5)
-        pass10 = self.predictor.get_next_pass(tortu1, date, aos_at_dg=10)
-        pass15 = self.predictor.get_next_pass(tortu1, date, aos_at_dg=15)
-        self.assertTrue(pass5.aos < pass10.aos < pass15.aos)
-        self.assertTrue(pass5.los > pass10.los > pass15.los)
-        self.assertTrue(pass5.max_elevation_deg == pass10.max_elevation_deg)
+        pass5 = self.predictor.get_next_pass(ARG, date, aos_at_dg=5)
+        pass10 = self.predictor.get_next_pass(ARG, date, aos_at_dg=10)
+        pass12 = self.predictor.get_next_pass(ARG, date, aos_at_dg=12)
```

Comment:
This pass has a maximum elevation of ~13, and therefore using `aos_at_dg=15` would select a different pass and break the `5 > 10 > 12` chain.